### ISSUE TITLE
The nextcloud program flashes back when renaming files

### ISSUE THREAD

**Issue by lsl061085** [nextcloud/ios#846]
Nextcloud for ios 2.23.3.4 ，When I perform the operation of renaming the file, the nextcloud program will automatically exit.

iphone6s：11.2.6
nextcloud server：15.0.7
php：7.2

**Comment by marinofaggiana** [comment]
Hi @lsl061085 ... no, strange, can you try with the next version or if you want create an accont test for me.

**Comment by lsl061085** [comment]
Hi,@marinofaggiana ，I created a test account, you can help with the test, thank you.
nextcloud server：https://cloud.wmysummer.com:2000
account: demo
password: demo123456

**Comment by marinofaggiana** [comment]
I have try but I don't have problem with the rename file ...

**Comment by lsl061085** [comment]
> I have try but I don't have problem with the rename file ...

Curiously,I reinstalled Nextcloud from the App Store,I quit all the background programs of IOS， but the same problem occurred.

**Comment by marinofaggiana** [comment]
@lsl061085 this issue with all files ? account demo included ?

**Comment by lsl061085** [comment]
> @ lsl061085这个问题与所有文件？包括帐户演示？

Yes, I recorded two test videos and uploaded them to demo account. You can watch them.
![image](https://user-images.githubusercontent.com/28956825/56981326-312e1d80-6bb1-11e9-871e-e5b2e6de073e.png)

**Comment by marinofaggiana** [comment]
Sorry but without a valid test , I can't find the problem.
